### ISSUE TITLE
[A11y] Advanced Search - inconsistent navigation - breadcrumbs

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1814]
### Operating System

Windows

### Browser

Edge

### Affected page

Advanced Search

### Admin user?

Yes

### Issue Description

Just noticed that some pages have the page name within breadcrumbs, but Advanced Search doesn't.  This could be a failure of navigation consistency (WCAG 3.2.3).  Probably worth fixing even if strictly for usability.

### Suggested Fix

Make sure all pages always identify themselves in the breadcrumbs.
